From 270462137cf7462108dc94f8b02c60ce099b2674 Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Tue, 3 Feb 2015 13:15:09 -0500
Subject: [PATCH] laird: ath6kl: Remove error output "temporary war to avoid
 sdio crc error"

This is just causing confusion from customers thinking this is indicative
of an issue, when it is not.

Bug: 6690
---
 drivers/net/wireless/ath/ath6kl/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath6kl/init.c b/drivers/net/wireless/ath/ath6kl/init.c
index 523f7cd..0a3c275 100644
--- a/drivers/net/wireless/ath/ath6kl/init.c
+++ b/drivers/net/wireless/ath/ath6kl/init.c
@@ -1484,7 +1484,7 @@ static int ath6kl_init_upload(struct ath6kl *ar)
 
 	/* WAR to avoid SDIO CRC err */
 	if (ar->hw.flags & ATH6KL_HW_SDIO_CRC_ERROR_WAR) {
-		ath6kl_err("temporary war to avoid sdio crc error\n");
+		/* ath6kl_err("temporary war to avoid sdio crc error\n"); */
 
 		param = 0x28;
 		address = GPIO_BASE_ADDRESS + GPIO_PIN9_ADDRESS;
-- 
1.9.1

